subscription-engine/kafka-ws/README.md

```diff
@@ -335,9 +338,12 @@ redis.host| Redis host | kafka-ws-redis | string | KAFKA_WS_REDIS_HOST
 redis.port | Redis port | 6379 | integer | KAFKA_WS_REDIS_PORT
 redis.db | Redis database | 1 | integer | KAFKA_WS_REDIS_DB
 redis.connect_timeout | Redis Connection timeout | 3600000 | integer | KAFKA_WS_REDIS_CONNECT__TIMEOUT
-redis.strategy.connect.after | Redis strategy connection timeout (ms) | 5000 | integer | KAFKA_WS_REDIS.RECONNECT.AFTER
+redis.strategy.connect.after | Redis strategy connection timeout (ms) | 5000 | integer | KAFKA_WS_REDIS_RECONNECT_AFTER
```

Comment:
Redis is being used in many places, so the Redis wrapper might be a candidate for be moved to the SDK.